src/pe_source/data/pe_db/config.py

```diff
@@ -82,3 +82,21 @@ def cybersix_token():
     }
     resp = requests.post(url, headers=headers, data=payload).json()
     return resp["access_token"]
+
+
+def get_params(section):
+    """Get data source parameters."""
+    if os.path.isfile(REPORT_DB_CONFIG):
+        parser = ConfigParser()
+        parser.read(REPORT_DB_CONFIG, encoding="utf-8")
+        if parser.has_section(section):
+            params = parser.items(section)
+        else:
+            raise Exception(
+                "Section {} not found in the {} file".format(section, REPORT_DB_CONFIG)
```

Comment:
Do we want to follow the same '%s' string replacement field methods like we do in the rest of the code?

Reply:
That's up to you.  The main thing is that logging statements use printf %-style formatting and not f-strings - see [here](https://blog.pilosus.org/posts/2020/01/24/python-f-strings-in-logging/) for an explanation.  For exceptions and pretty much everywhere else, I generally prefer to use f-strings.